ansible-scylla-node/defaults/main.yml

```diff
@@ -12,6 +88,9 @@ scylla_nic: "{{ ansible_default_ipv4.interface }}"
 install_type: online
 
 # Defines if the node firewall should be enabled or not
+# Values:
+#  [default] false: all front-end firewall services will be disabled and iptables rules flushed
+#  true: keep node configuration as is
```

Comment:
just a nitpick, commits should be self isolated and we shouldn't mix features / changes to them
so next time this change has to be a separate PR,  since it has nothing to do with feature/upgrade

Reply:
In my opinion, this role should not interfere with the firewall at all. I would prefer if it wouldn't touch it so another role can manage it. There are many different firewall daemons used, and it shouldn't be adding complexity to this role. It can be handled like mdadm and swap is handled.